### PR TITLE
💍 Add tests for SQFormIconButton

### PR DESCRIPTION
**Pull request by laurelbean**
Added some tests for SQFormIconButton. These are generally the same as the tests for SQFormButton since they have similar functions.

✅ Closes: #221

### CODE REVIEW

**Review by codyarose — CHANGES_REQUESTED** [review]
Looks good - I think the only thing that's missing is a test to ensure that an icon is rendered within the button. Probably just need to check that the SVG element is there - and maybe that the SVG's `title` attribute matches the name of whichever Mui icon is specified, that may not be possible or necessary though

File: stories/__tests__/SQFormIconButton.stories.test.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+      const iconButton = screen.getByRole('button', {
+        name: /form submission/i
+      });
+
+      expect(iconButton).toBeDisabled();
+    });
+  });
+
+  describe('Button with Test Field', () => {
```
Comment (20BBrown14) [inline stories/__tests__/SQFormIconButton.stories.test.js:105]: typo: `text`

**Review by 20BBrown14 — APPROVED** [review]

File: stories/__tests__/SQFormIconButton.stories.test.js (hunk 1 above)
Comment (laurelbean) [reply stories/__tests__/SQFormIconButton.stories.test.js:105]: I meant it to be Test. The name of the story is `WithTestField`

File: stories/__tests__/SQFormIconButton.stories.test.js [hunk 2]
```diff
@@ -12,12 +12,23 @@ const {
...
 describe('SQFormIconButton Tests', () => {
   describe('Button Only', () => {
+    it('should render a button with an icon', () => {
+      render(<SQFormIconButton exampleIcons={CheckCircle} />);
+
+      const iconButton = screen.getByRole('button', /form submission/i);
+      const svg = document.getElementsByTagName('svg');
+
+      expect(iconButton).toBeInTheDocument();
+      expect(svg.length).toEqual(1);
```
Comment (laurelbean) [inline stories/__tests__/SQFormIconButton.stories.test.js:22]: @codyarose This is the closest I could get it to test for the icon. The <svg> is `area-hidden`, doesn't have a role, and doesn't have any text.

Comment (SeanGroff) [reply stories/__tests__/SQFormIconButton.stories.test.js:24]: @laurelbean @codyarose I'm going to merge this PR, then i'll add a comment to #228 that we should fix this test as part of the PR that resolves issue #228

**Review by SeanGroff — APPROVED** [review]

**Pull request merged**